src/tests/rnp_tests.h

```diff
@@ -62,6 +62,8 @@ void rnpkeys_generatekey_verifykeyHomeDirOption(void **state);
 
 void rnpkeys_generatekey_verifykeyKBXHomeDirOption(void **state);
 
+void rnpkeys_generatekey_verifykeyNonexistingHomeDir(void **state);
+
```

Comment:
This looks like rebase-introduced artifact, as this test was removed some time ago.